src/lib.rs

```diff
@@ -103,9 +117,32 @@ impl Color {
             r: r,
             g: g,
             b: b,
-            _a: a,
+            a: a,
         }
     }
+
+    // Compute a canonical color that is equivalent to the input color
+    // for preblend table lookups.
+    pub fn quantize(&self) -> Color {
```

Comment:
given that the values are quantized now, the type should be different. Perhaps, `QuantizedColor`?